Makefile

```diff
@@ -10,31 +13,44 @@ compile:
 image: compile
 	operator-sdk build $(IMAGE_TAG)
 
+.PHONY: test
+test:
+	@echo $(CRDS)
```

Comment:
Not sure if you meant to leave this target in.

Reply:
Gone.